run-test.sh

```diff
@@ -257,6 +264,11 @@ then
     CoreFxBins="$ProjectRoot/bin/$OS.AnyCPU.$Configuration"
 fi
 
+if [ "$CoreFxNativeBins" == "" ]
+then
+    CoreFxNativeBins="$ProjectRoot/bin/$OS.x64.$Configuration"
```

Comment:
The default should have /Native at te en as that's where they are built locally.